test/context.test.py

```diff
@@ -204,7 +204,7 @@ def test_context_delete_unset_after_removal(self):
         """Test that context is unset if its definition has been removed."""
         self.t('context define work project:Work', input='y\ny\n')
         self.t('context work')
-        code, out, err = self.t('context delete work', input='y\n\y\n')
+        code, out, err = self.t('context delete work', input='y\n\\y\n')
```

Comment:
```suggestion
        code, out, err = self.t('context delete work', input='y\ny\n')
```
This is similar to line 205. Interesting that the `y` `\y` input also works!